content/sv/getting-started.md

```diff
@@ -30,7 +30,7 @@ ACME-klienten [Certbot]. Den kan automatisera certifikatutfärdande och
 installation utan nertid. Den har även expertlägen för de som inte föredrar
 automatisk konfiguration. Den är lätt att använda, fungerar på många
 operativsystem och har väldigt bra dokumentation. [Besök
-Certbot-sajten][Certbot] (engelska) för att få skräddarsydda instruktioner för
+Certbot-sajten][Certbot](engelska) för att få skräddarsydda instruktioner för
```

Comment:
I think this change was a mistake. The space is needed. It has nothing to do with Markdown.

Reply:
I'm sorry, you're right, do you think you have time to open a PR to fix it?

Reply:
Sure, I'll do it.

Reply:
Done as part of pull request #989.